Documentation/security/network/encryption-ipsec.rst

```diff
@@ -184,6 +184,14 @@ Key Rotation
    is, all nodes in the cluster (or clustermesh) should be on the same Cilium
    version before rotating keys.
 
+.. attention::
+
+   Key rotations do not allow changes in algorithms that involve different
+   authentication key lengths. If this is attempted, Cilium will reject
+   the new key to prevent potential connectivity disruptions, and log an
+   error message. Upon restarting the agent, Cilium will configure itself
+   using the latest available key.
```

Comment:
This is ambiguous. First sentence says it's not allowed; last sentence explains how it will work :sweat_smile: Could we reformulate to say it's "not recommended" and then proceed to explain the behavior if it is still done. "reject" is also not the correct word here IMO, as the agent doesn't so much reject the new key as it delays its application.

Reply:
Many thanks for the feedback, just pushed the new sentence :pray:

Reply:
:ok_hand: